test/unix/test_lwt_bytes.ml

```diff
@@ -732,55 +739,76 @@ let suite = suite "lwt_bytes" [
 
     test "mincore buffer length = page_size * 2, n_states = 1"
       ~only_if:(fun () -> not Sys.win32) begin fun () ->
-      test_mincore (Lwt_bytes.page_size * 2) Lwt_bytes.page_size 1
-      >>= fun states ->
-      Lwt.return (states.(0) = true)
+      Lwt.catch
+        (fun () ->
+           test_mincore (Lwt_bytes.page_size * 2) Lwt_bytes.page_size 1
+           >>= fun () -> Lwt.return true
+        )
+        (function
```

Comment:
For the tests that we don't expect to raise an exception, I suggest not checking for it explicitly. The test framework will report it if it occurs. As an additional benefit, this will let us know if a different exception occurs. So, I think this test should be how it was, except for the new unconditional `Lwt.return true` at the end.

Reply:
ok